manifests/base/manager/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
Can we keep name of this file `manager.yaml` and include Deployment + Service in this file ?
I don't think it is necessary to distinguish it.